### ISSUE TITLE
Faye message Ids

### ISSUE THREAD

**Issue by amirvt** [wied03/faye_dotnet_client#1]
Hi I realize this project is no longer maintained, but I want to leave this here for any future visitors who might run into the same problem I had.

Anyway, Faye message ids are not integers. This causes the library to randomly crash with number format exceptions.
Here's an example message I captured with the [CometD.NET](https://github.com/Oyatel/CometD.NET) library.

```
{"channel":"/chan/name","data":"{\"hello\":\"hello\"}","id":"f5"}
```

As you can see the id is not an integer but this library assumes they are.

**Comment by wied03** [comment]
Perhaps I can get time to fix that. The project I was going to use this for
never panned out.

sent from my mobile device

**Comment by amirvt** [comment]
Thanks for the response.
It's a shame your project never panned out.

**Comment by wied03** [comment]
@amirvt are you actively using this?

**Comment by amirvt** [comment]
No, but I did try it. I needed a .NET client and this was the first project that showed up on google after I searched "c# faye client". It worked initially but then it started giving number format exceptions.
Right now I'm using [CometD.NET](https://github.com/Oyatel/CometD.NET). Its not ideal but it got the job done.
